packages/@jsii/spec/lib/assembly.ts

```diff
@@ -143,6 +143,13 @@ export interface Assembly extends AssemblyConfiguration, Documentable {
    * @default none
    */
   readme?: { markdown: string };
+
+  /**
+   * List of bin-scripts
+   *
+   * @default none
+   */
+  bin?: { [script: string]: string };
```

Comment:
Let's tighten this one up from the start (cannot fix the older fields to `readonly` because... backwards-compatibility... but we can be nice with the new ones!)

```suggestion
  bin?: { readonly [script: string]: string };
```